virttest/remote.py

```diff
@@ -1351,3 +1354,91 @@ def run(self, command, timeout=60, ignore_status=False):
         if status and (not ignore_status):
             raise process.CmdError(command, cmd_result)
         return cmd_result
+
+
+class VMManager(object):
```

Comment:
it sounds like class RemoteVMManager name is more explicit than VMManager  according to below comment "Manage VM on remote host"